### ISSUE TITLE
Stop criteria for no-cursor trial

### ISSUE THREAD

**Issue by thartbm** [thartbm/PyVMEC#28]
In no-cursor trials, the trial seems to end whenever people are further than half the target distance away from home. However, there should be an additional criteria: very low speed. In the Matlab VMEC this is defined like this:

                        if (VBLtime - 0.5) > min(trialdata.VBLtime)
                            idx = find(trialdata.VBLtime > (VBLtime - 0.5));
                            if sum(sqrt(diff(trialdata.Xmouse(idx)).^2 + diff(trialdata.Ymouse(idx)).^2)) < numel(idx)
                                out = true;
                            end;
                        end;

So basically, the participant has to have moved for half a second, and then across the last half a second, the total movement allowed is 1 pixel per sample. That's when the 'out' variable is set to True, which indicates that the outward reach is done, and the participant now has to move back to the home position.

I'm thinking that we want to reduce the time we consider for this to 300 ms.
